HelloWorldServlet/src/main/java/com/srccodes/example/HelloWorld.java

```diff
@@ -0,0 +1,50 @@
+package com.srccodes.example;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+ * Servlet implementation class HelloWorld
+ 
+@WebServlet("/HelloWorld")
+public class HelloWorld extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+      
+     * @see HttpServlet#HttpServlet()
+    public HelloWorld() {
+        super();
+    }
+
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		response.setContentType("text/html");
+        PrintWriter printWriter = response.getWriter();
+        printWriter.println("<h1>Hello World!</h1>");
+	}
+
+	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Delete this *** @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)**